inst/ms/aggreCAT.qmd

```diff
@@ -903,17 +982,16 @@ using default priors, priors are generated from a predictive model that
 estimates the probability of a claim replicating based on
 characteristics of the claim and publication [@Gould2021a]. Priors are
 parsed as supplementary data to the wrapper function
-[BayesianWAgg]{.fct} using the argument [priors]{.arg} (section
-@sec-bayesian-supplementary-data ) with each claim having its own unique
+[BayesianWAgg]{.fct} using the argument [priors]{.arg} (@sec-bayesian-supplementary-data ) with each claim having its own unique
 prior.
 
 We illustrate aggregation of participant judgements using the method
 `BayTriVar` to generate a Confidence Score for the claim `czttvy`. Note
 that [BayesianWAgg]{.fct} expects best estimates in the form of
 probabilities, so to convert elicited values in the form of percentages
-within the data parsed to [expert\\_judgements]{.arg} to probabilities,
+within the data parsed to [expert\_judgements]{.arg} to probabilities,
```

Comment:
noticing now that expert_judgements doesn't seem to be in italics or code font throughout the manuscript

Reply:
`[expert_judgements]{.arg}` syntax not working for some reason, replace with `` `expert_judgements` ``